mmaction/datasets/pipelines/loading.py

```diff
@@ -4,6 +4,7 @@
 import shutil
 import warnings
 
+import librosa
```

Comment:
make this dependency optional

Reply:
And how about make it inside the `MultiModalityDecode` class ?

Reply:
`MultiModalityDecode` is more like a an experimental thing, which decode rgb+flow+audio at the same __call__ at(around) same frame_idx. It will be removed later. Let's just add import librosa in AudioDecode __init__ and wrap with `try`.